workflow-service-sdk/pom.xml

```diff
@@ -27,10 +28,15 @@
             <plugin>
                 <groupId>io.spring.javaformat</groupId>
                 <artifactId>spring-javaformat-maven-plugin</artifactId>
-                <version>0.0.34</version>
-                <configuration>
-                    <skip>true</skip>
-                </configuration>
+                <version>${spring.javaformat.version}</version>
+                <executions>
+                    <execution>
+                        <phase>verify</phase>
```

Comment:
I think the phase is wrong here.

We generate the files at the default phase.
`verify` phase is invoked after the sources are already compiled and packaged.
This means the content of the jar will differ from the actual source.

I suggest being explicit of both:
generate-sources - for generating the sources
process-resources - for formatting the sources

see https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#default-lifecycle

Reply:
Done, thanks!